manifests/g/GitHub/cli/2.63.0/GitHub.cli.installer.yaml

```
@@ -14,22 +14,22 @@ InstallModes:
 UpgradeBehavior: install
 Commands:
 - gh
+ReleaseDate: 2024-11-27
 ElevationRequirement: elevatesSelf
 Installers:
 - Architecture: x64
   InstallerUrl: https://github.com/cli/cli/releases/download/v2.63.0/gh_2.63.0_windows_amd64.msi
   InstallerSha256: E45595E9B416EA0933AA1FAAEDCBEA14C81D5512DA342C0219A4541CED76DCF7
   ProductCode: '{FAAF4306-25F7-422D-9758-CDFD6E6FC729}'
   AppsAndFeaturesEntries:
-  - ProductCode: '{C08A19D6-620B-415D-B4FA-0224B3D6F78E}'
+  - ProductCode: '{FAAF4306-25F7-422D-9758-CDFD6E6FC729}'
     UpgradeCode: '{8CFB9531-B959-4E1B-AA2E-4AF0FFCC4AF4}'
 - Architecture: x86
   InstallerUrl: https://github.com/cli/cli/releases/download/v2.63.0/gh_2.63.0_windows_386.msi
   InstallerSha256: 9F9324FDD2576B2289079CADE9D5A74CF7D8B80FFF64D9C8C3BFEADD44C2DDD0
   ProductCode: '{52F2E2D9-E7E7-48AE-B12A-875739D6BB61}'
   AppsAndFeaturesEntries:
-  - ProductCode: '{8937394A-B572-4DB3-9ECC-CD82153E3754}'
+  - ProductCode: '{52F2E2D9-E7E7-48AE-B12A-875739D6BB61}'
     UpgradeCode: '{767EC5D2-C8F0-4912-9901-45E21F59A284}'

```

```suggestion
  - UpgradeCode: '{767EC5D2-C8F0-4912-9901-45E21F59A284}'
```

Remove ProductCode from ARP level entirely. Some manifest creator tools do not update this automatically leaving old values getting carried forward in manifests and then having to make manual PRs to fix. It's defined with correct values in the Installer level which suffices.